content/general/authentication.textile

```diff
@@ -207,6 +170,24 @@ ably.auth.requestToken({ clientId: 'client@example.com' }, function(err, tokenDe
+
+The table below indicates which feature is best given a certain requirement:
+
+|_. Condition |_. Basic |_. Token |_. Description |
+| Your scripts are exposed | <div style="text-align: center;"><img src="/images/table-images/cross.svg"></div> | <div style="text-align: center;"><img src="/images/table-images/tick.svg"></div> | If the script, program or system holding the key is exposed, for example on a user's device, you should not embed an API key and instead use Token Authentication. If the script is on a secure environment such as your own server, an API key with Basic Authentication is fine. |
+| Your connection is insecure | <div style="text-align: center;"><img src="/images/table-images/cross.svg"></div> | <div style="text-align: center;"><img src="/images/table-images/tick.svg"></div> | If there is a risk of exposure of the client’s credentials, either directly or over an insecure, or insecurely proxied, connection, Token Authentication should be used. If you are sure the connection is secure and unmediated, Basic Authentication is acceptable. |
+| You have no server to control access | <div style="text-align: center;"><img src="/images/table-images/tick.svg"></div> | <div style="text-align: center;"><img src="/images/table-images/cross.svg"></div>  | If you do not have your own server to perform authentication and provide tokens to users, you'll need to use Basic Authentication. |
+| You require fine-grained access control | <div style="text-align: center;"><img src="/images/table-images/cross.svg"></div> | <div style="text-align: center;"><img src="/images/table-images/tick.svg"></div> | If you need to provide privileges on a user-by-user basis, you'd be better using Token Authentication. If you only need a few access control groups, Basic Authentication is reasonable. |
+| Users need restricted periods of access | <div style="text-align: center;"><img src="/images/table-images/cross.svg"></div> | <div style="text-align: center;"><img src="/images/table-images/tick.svg"></div> | If you need users to only have access for a certain period of time, or the ability to revoke access, Token Authentication is needed. If users are able to always have access, Basic Authentication is acceptable. |
+| Users need to indentify themselves | <div style="text-align: center;"><img src="/images/table-images/exclamation.png"></div>  | <div style="text-align: center;"><img src="/images/table-images/tick.svg"></div> | If the user can be trusted to identify itself, Basic Authentication is fine. If the user cannot be trusted however, Token Authentication is better as it allows for a trusted token distributor to identify the user instead. |
```

Comment:
indentify = identify - Typo. Perhaps use less strong wording in the 'condition' section say instead of 'Your scripts are exposed', make it more subtle and say 'Your scripts might be exposed' - something that doesn't sound too much like pointing out their code etc?

Also, probably call it 'Scenario' instead of a 'Condition'?